internal/sirius/document.go

```diff
@@ -27,14 +29,24 @@ const (
 	TypeSave    string = "Save"
 )
 
-func (c *Client) Documents(ctx Context, caseType CaseType, caseId int, docType string) ([]Document, error) {
+func (c *Client) Documents(ctx Context, caseType CaseType, caseId int, docTypes []string, notDocTypes []string) ([]Document, error) {
 	var d []Document
 
 	if caseType == CaseTypeDigitalLpa {
 		caseType = CaseTypeLpa
 	}
 
-	url := fmt.Sprintf("/lpa-api/v1/%s/%d/documents?type[]=%s", caseType+"s", caseId, docType)
+	query := ""
+
+	for _, docType := range docTypes {
+		query = query + "&type[]=" + docType
+	}
+
+	for _, docType := range notDocTypes {
+		query = query + "&type[-][]=" + docType
+	}
+
+	url := fmt.Sprintf("/lpa-api/v1/%s/%d/documents?%s", caseType+"s", caseId, strings.Trim(query, "&"))
```

Comment:
how come we have to specify which types of documents we want and also don't want, does just using one or the other not work correctly?

Reply:
Outgoing documents are either Draft, Preview or Save, which is easy to manage, but incoming documents can have anything as their type.

So when we're getting a list of drafts, we want to do "is Draft". When we're displaying the list of "published" documents, it's "is not Draft nor Preview".

Really I think we shouldn't be mixing incoming and outgoing like this, but we're obviously way past that decision now 😅